kevm-pyk/src/kevm_pyk/kproj/evm-semantics/evm-types.md

```diff
@@ -385,6 +390,7 @@ Accounts
 
 ```k
     syntax Account ::= ".Account" | Int
+    syntax AccountNotNil ::= Int
```

Comment:
Don't create a new sort here. If you really like having the descriptive name, replace "::=" with "=" so it just creates an alias.

Reply:
Alright!

Reply:
This is still unresolved.

Reply:
Just fixed, sorry!